records/generator_test.go

```diff
@@ -272,6 +272,14 @@ func TestInsertState(t *testing.T) {
 		t.Errorf("should return the container IPs 10.3.0.1, 10.3.0.2 for the task, but got %v", rrs)
 	}
 
+	rrs, ok = rg.As["_container.reviewbot.marathon.mesos."]
+	if !ok {
+		t.Error("should find this running task - A record")
```

Comment:
Can we have a better error message here? I know the rest of the test is no example, but if we keep improving the field when we touch it, we'll be in very good shape very soon :)